src/main/java/com/minecolonies/coremod/util/Utils.java

```diff
@@ -434,4 +438,55 @@ public static int getFortuneOf(@Nullable final ItemStack tool)
         }
         return fortune;
     }
+    
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)